### PR TITLE
Fix Storage Files DataLake and Storage Blobs ChangeFeed issues about Readme and Samples 

### PR DESCRIPTION
**Pull request by Tianyun-G**
Fix issues https://github.com/Azure/azure-sdk-for-net/issues/13633 https://github.com/Azure/azure-sdk-for-net/issues/13634

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md [hunk 1]
```diff
@@ -107,14 +107,14 @@ await foreach (BlobChangeFeedEvent changeFeedEvent in changeFeedClient.GetChange
 ## Troubleshooting
 All Blob service operations will throw a
 [RequestFailedException][RequestFailedException] on failure with
-helpful [`ErrorCode`s][error_codes].  Many of these errors are recoverable.
+helpful [`ErrorCode`][error_codes].  Many of these errors are recoverable.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md:110]: As Ted pointed out [here](https://github.com/Azure/azure-sdk-for-net/issues/13633#issuecomment-668761384), the s should still be here.

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md [hunk 2]
```diff
@@ -107,14 +107,14 @@ await foreach (BlobChangeFeedEvent changeFeedEvent in changeFeedClient.GetChange
...
 
 ## Next steps
 
 Get started with our [Change Feed samples][samples]:
 
 1. [Hello World](samples/Sample01a_HelloWorld.cs): Get changes that have occured in your storage account (or [asynchronously](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.ChangeFeed/samples/Sample01b_HelloWorldAsync.cs))
-2. [Auth](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs): Authenticate with connection strings, public access, shared keys, shared access signatures, and Azure Active Directory.
+2. [Auth](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs.ChangeFeed/samples/Sample02_Auth.cs): Authenticate with connection strings, public access, shared keys, shared access signatures, and Azure Active Directory.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md:117]: This is a broken link now. This was supposed to point to the blob auth sample

Comment (amnguye) [reply sdk/storage/Azure.Storage.Blobs.ChangeFeed/README.md:117]: I see the added file now. my bad

**Comment by amnguye** [comment]
Hm. Looks like CI wont pass because the link you referenced gives a 404, even though it's getting added in this PR. Maybe we should divide this PR, and merge the sample first and then the readme next.

**Comment by Tianyun-G** [comment]
@amnguye - I have divided this PR, this is another PR https://github.com/Azure/azure-sdk-for-net/pull/15083 about adding Sample02_Auth.cs.

**Review by amnguye — APPROVED** [review]

**Comment by jsquire** [comment]
Hi @Tianyun-G.  There hasn't been recent engagement on this PR.  Would you please be so kind as to let us know if this is still an active work stream or if the PR should be closed out?

**Comment by Tianyun-G** [comment]
@jsquire This PR is still an active work stream. We are waiting for it to be merged.

**Comment by jsquire** [comment]
> @jsquire - This PR is still an active work stream. We are waiting for it to be merged.

Thank you for the update.   Is there a specific person that we're waiting on or reason that it has not been merged?

**Comment by Tianyun-G** [comment]
@jsquire - We don't have the permission to merge. Can you help us merge it?

**Comment by jsquire** [comment]
> @jsquire - We don't have the permission to merge. Can you help us merge it?

Sure, I can do that.

**Pull request merged**